src/Order/OrdersList.php

```diff
@@ -41,6 +41,12 @@ public function init_hooks() {
 
 		// add 'Label Created' orders page column content
 		add_action( 'manage_shop_order_posts_custom_column', array( $this, 'add_order_barcode_column_content' ), 10, 2 );
+
+		// Make delivery date column sortable.
+		add_filter( "manage_edit-shop_order_sortable_columns", array( $this, 'sort_delivery_date_column' ) );
```

Comment:
```suggestion
		add_filter( 'manage_edit-shop_order_sortable_columns', array( $this, 'sort_delivery_date_column' ) );
```
Let's standardize to use single quote if we don't really need double quote.